cellrank/tl/_lineage.py

```diff
@@ -706,6 +706,22 @@ def copy(self, _="C") -> "Lineage":
     def __copy__(self):
         return self.copy()
 
+    @property
+    def priming_degree(self) -> np.ndarray:
+        """Degree of priming as described in [Velten17]_."""
+        probs = self.X
+        return np.nan_to_num(
+            np.sum(probs * np.log2(probs / np.mean(probs, axis=0)), axis=1),
```

Comment:
@Marius1311 2 questions, both regarding:
Do you refer to this?
> clusters: a list of "early" clusters that are used to define the "average" lineage probabilities when using kl_divergence. By default, all cells are considered

Reply:
Also, I'd consider passing a vector of cell ids instead of cluster (more general, easier to implement - no need for cluster_key + checks).